src/CMakeLists.txt

```diff
@@ -391,6 +396,7 @@ target_link_options(runblack-reassembled
         "/FIXED"
         "/sectionvsize:.text,0x004a795d"
         "/sectionvsize:.data,0x5f9e00"
+        "/sectionvsize:.rsrc,0x393E"
```

Comment:
This size should not be explicitly stated in the linker if the reversing is done correctly.
Additionally, there should be no more post processing except for the garbage license string.